econml/tests/test_federated_learning.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) PyWhy contributors. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+import unittest
+from econml.dml import LinearDML
+from econml.inference import StatsModelsInference
+from econml.sklearn_extensions.federated_learning import FederatedLearner
+
+
+class FunctionRegressor:
+    def __init__(self, func):
```

Comment:
Add a docstring just to make it even more clear what's happening here.

```suggestion
    """A simple model that ignores the data it is fitted on, always just using the specified function to predict"""
    def __init__(self, func):
```

Reply:
Agreed, updated.